firebase-vscode/src/core/webhook.ts

```diff
@@ -4,13 +4,20 @@ import { createServer } from "http";
 import bodyParser from "body-parser";
 import { VSCODE_MESSAGE, WebhookBody } from "../../../src/dataconnect/webhook";
 import { pluginLogger } from "../logger-wrapper";
-export function registerWebhooks() {
+import { findOpenPort } from "../utils/port_utils";
+import { setTerminalEnvVars } from "../data-connect/terminal";
+export async function registerWebhooks() {
   const app = express();
   app.use(bodyParser.json()); // for parsing application/json
 
   const server = createServer(app);
-  const port = 40001; // TODO: generate port dynamically with this as default, and pass as env var to integrated terminal
+  const DEFAULT_PORT = 40001;
```

Comment:
UPPER_CASE is used for file or global constants. Either move outside of this function  or just call this defaultPort.